### PR TITLE
Billing Platform - follow up and TODOs

### PR DESCRIPTION
**Pull request by ClaudeROSSI**

### CODE REVIEW

File: src/app/pages/settings-integration/billing/settings-billing.component.ts [hunk 1]
```diff
@@ -62,7 +62,11 @@ export class SettingsBillingComponent implements OnInit {
       this.checkConnectionContext(settings);
       // Init form
       this.formGroup.markAsPristine();
-      this.billingAccountTableDataSource.loadData().subscribe();
+      if ( this.isConnectedAccountActive ) {
+        // TO DO - This should not be there -move it in the nested component
+        // TO DO - This is called twice when navigating to the billing settings
+        this.billingAccountTableDataSource.loadData().subscribe();
```
Comment (ClaudeROSSI) [inline src/app/pages/settings-integration/billing/settings-billing.component.ts:68]: @RohanAntony : Please take over

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -1752,7 +1752,7 @@
       "connected_account": {
         "user": "User",
         "status": "Status",
-        "title": "Connected Accounts",
+        "title": "Connected Accounts - TODO - move account-related texts in a dedicated section (like the invoices section)",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/en.json:1755]: @RohanAntony - please move the texts as mentioned here. Let's sync when you have some time

**Review by ClaudeROSSI — COMMENTED** [review]
@RohanAntony - please have a look to my comments

File: src/assets/i18n/en.json (hunk 2 above)
Comment (RohanAntony) [reply src/assets/i18n/en.json:1755]: Something like this ? 

image

**Comment by RohanAntony** [comment]
@ClaudeROSSI I can't seem to add you as a reviewer because you created the PR. Please take a look at the changes.

File: src/assets/i18n/cs.json [hunk 3]
```diff
@@ -1517,6 +1517,26 @@
...
+      "create_error": "Error occurred while adding the account, check the logs",
+      "onboard_success": "The onboarding process has been triggered. A mail has been sent to the business owner",
+      "onboard_error": "Error occurred while onboarding the account, check the logs"
+    },
+    "status": {
+      "account_idle": "Idle",
+      "account_pending": "Pending",
+      "account_active": "Active"
+    }
+  },
```
[the remark this answers is not in the document]
Comment (ClaudeROSSI) [reply src/assets/i18n/cs.json:1539]: yes ... this is what I want
please also do it for the "add" button and its tooltip

File: src/app/pages/accounts/accounts.component.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+export class AccountsComponent{
+
+  // eslint-disable-next-line no-useless-constructor
+  public constructor(
+    public billingAccountTableDataSource: BillingAccountsTableDataSource,
+    public translateService: TranslateService
+  ) {
+  }
+
+}
```
[the remark this answers is not in the document]
Comment (ClaudeROSSI) [reply src/app/pages/accounts/accounts.component.ts:19]: > Added a seperate component but the module wasn't created. We can create an independent module when it gets converted to a page or should we create an independent module now? There is also a question of routing file which is also added if its an independent page.

I did a minor change and it now loads properly ... please pull my changes!

File: src/app/pages/accounts/account/account-dialog.component.html [hunk 5]
```diff
@@ -4,7 +4,7 @@
       <mat-tab>
         <ng-template mat-tab-label>
           <mat-icon>business</mat-icon>
-          <span>{{ "settings.billing.connected_account.create" | translate }}</span>
+          <span>{{ "accounts.list.add" | translate }}</span>
```
Comment (ClaudeROSSI) [inline src/app/pages/accounts/account/account-dialog.component.html:7]: accounts.list.add_account! please

File: src/assets/i18n/de.json [hunk 6]
```diff
@@ -1517,6 +1517,26 @@
     ],
     "logo_size_error": "Die maximale Bildgröße wurde überschritten, das Bild muss kleiner als {{maxPictureKb}} Kb sein "
   },
+  "accounts": {
+    "title": "Connected Accounts",
+    "list": {
+      "add": "Add connected account",
+      "business_owner": "Business Owner",
+      "status": "Status",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/de.json:1525]: account_add
account_status

**Review by ClaudeROSSI — COMMENTED** [review]
Only a few comments - take care, thanks

**Review by RohanAntony — COMMENTED** [review]
Adding changes

**Review by RohanAntony — APPROVED** [review]

**Pull request merged**